README.md

```diff
@@ -135,11 +137,9 @@ void main() {
 
 Proper usage of refs here is a little bit different from usage in react. You can specify
 a ref name in component props and then call ref method to get the referenced element.
-Return value for dart components and for javascript components is different.
-For dart component, you get an instance of dart class of the component.
-For js components (like DOM elements), you get instance of jsObject representing the
```

Comment:
@greglittlefield-wf Where someone is using a non-Dart composite component? I suppose so, I'll add it back.